src/platform_impl/linux/event_loop.rs

```diff
@@ -760,13 +755,6 @@ impl<T: 'static> EventLoop<T> {
               Inhibit(false)
             });
           }
-          WindowRequest::Redraw => {
-            if let Err(e) = draw_tx.send(id) {
-              log::warn!("Failed to send redraw event to event channel: {}", e);
-            }
-
-            window.queue_draw();
```

Comment:
this is seems to be removed completely, is that desired?

Reply:
This is unintentional. But since we now understand `queue_draw` is more like a signal for gtk widgets to draw, I would like to think about if we still need this. `RequestRedraw` event is more for users to draw on GL library themselves IMHO.
I'll give my conclusion soon.

Reply:
Alright, I really think `queue_draw` could be removed. Calling `queue_draw` will invoke `connect_draw` and hence send another draw event. It doesn't make sense to send two draw events when calling `request_redraw`.
And for regular gtk widgets, they should handle all drawing processes properly.

Reply:
alright, sounds good